### PR TITLE
Refactored FormCarter for testing, added phpunit dependency and basic…

### PR DESCRIPTION
**Pull request by banasmat**
- Replaced usage of 'echo' with exceptions and return statements in FormCarter class.
- Removed instantiating Swift_Mailer outside of the class. It's now injected through constructor.
- Removed json response wrapper and header functions outside of the class. It shouldn't be responsible for the 'view'.
These changes made testing the class possible.
Added phpunit dependency with basic tests and info how to run them.

### CODE REVIEW

File: st-errors.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+2019-10-25 18:20:37: Connection could not be established with host smtp.example.com [php_network_getaddresses: getaddrinfo failed: No such host is known.  #0]
```
Comment (emtei) [inline st-errors.txt]: Should we gitignore this file?

**Review by emtei — CHANGES_REQUESTED** [review]

File: st-errors.txt (hunk 1 above)
Comment (IgorPodlawski) [reply st-errors.txt]: It's already in .gitignore after PR #4

**Review by kamiljedrkiewicz — APPROVED** [review]
Thanks, great job.

File: st-errors.txt (hunk 1 above)
Comment (emtei) [reply st-errors.txt]: So it needs to be removed from this PR

**Review by emtei — APPROVED** [review]

**Review by emtei — APPROVED** [review]

**Pull request merged**